test/passing/tests/break_separators-before_docked_wrap.ml.ref

```diff
@@ -98,6 +98,15 @@ let _ =
       0
   | _ -> 1
 
+let _ =
+  match something with
+  | [:
+     very_very_long_field_name_running_out_of_space
+   ; another_very_very_long_field_name_running_out_of_space; _
```

Comment:
As with `test/passing/tests/break_separators-after.ml.ref`, but this time the "_wrap" part of the settings means that we're doing flow-based wrapping, I think.  Not resolving in case you want to look at other parts of this file.